File: corr_en.py
import numpy as np

data = np.loadtxt('values.txt', skiprows=1)
nd = data.shape[0]
nv = data.shape[1]
covmx = np.cov(data.T)
v = ['x','y']

print("Covariance matrix from Monte Carlo simulation")
print(" number of data: {:d}".format(nd))
print("covariance matrix:")
for i in range(nv):
    for j in range(nv):
        print("{0:2s}{1:2s}: {2:8.4e}".format(v[i],v[j],covmx[i,j]))

File: corr_en.m
% Load data, skipping the first row
data = dlmread('values.txt', '', 1, 0);

% Number of data points (rows) and variables (columns)
[nd, nv] = size(data);

% Covariance matrix
covmx = cov(data);

% Variable labels
v = {'x', 'y'};

% Display results
fprintf('Covariance matrix from Monte Carlo simulation\n');
fprintf(' number of data: %d\n', nd);
fprintf('covariance matrix:\n');

for i = 1:nv
    for j = 1:nv
        fprintf('%2s%2s: %8.4e\n', v{i}, v{j}, covmx(i,j));
    end
end




